src/index.html

```diff
@@ -2,12 +2,57 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <card
+    class="card"
+    data-qa="card"
+    >
+      <img
+      src="../src/images/imac.jpeg"
+      alt="imac"
+      class="card__photo"
+      >
+      <section
+      class="card__description">
+        <p>
+        APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+        </p>
+      </section>
+      <section class="card__product-code">
+        <p>
+          Product code: 195434
+        </p>
+      </section>
+      <ul class="stars-layout">
+        <ul class="stars-layout__stars">
+          <li class="stars-layout__star"></li>
+          <li class="stars-layout__star"></li>
+          <li class="stars-layout__star"></li>
+          <li class="stars-layout__star"></li>
+          <li class="stars-layout__star"></li>
+        </ul>
+        <ul>
```

Comment:
think that element should be just a simple paragraph, not need to wrap it in the list, so remove redundant `ul `tag:

`<p class="stars-layout__text">Review: 5</p>`